Tests/ARTFallbackTest.m

```diff
@@ -43,4 +43,35 @@ - (void)testAllHostsIncludedOnce {
     }    
 }
 
+- (void)testCustomFallbackHosts {
+    NSArray *customHosts = @[@"test.ably.com", @"test2.ably.com", @"test3.ably.com"];
+    ARTFallback *f = [[ARTFallback alloc] initWithFallbackHosts:customHosts];
+    
+    NSSet *customSet = [NSSet setWithArray:customHosts];
+    NSMutableArray *hostsRandomised = [NSMutableArray array];
+    for(int i=0;i < [customHosts count]; i++) {
+        [hostsRandomised addObject:[f popFallbackHost]];
+    }
+    
+    //popping after all hosts are exhausted returns nil
+    XCTAssertTrue([f popFallbackHost] == nil);
+    
+    // all fallback hosts are used in artfallback
+    XCTAssertEqual([hostsRandomised count], [customHosts count]);
+    bool inOrder = true;
+    for(int i=0;i < [customHosts count]; i++) {
+        if(![[customHosts objectAtIndex:i] isEqualToString:[hostsRandomised objectAtIndex:i]]) {
+            inOrder = false;
+            break;
+        }
+    }
+    //check artfallback randomises the order.
+    XCTAssertFalse(inOrder);
```

Comment:
Good point! BTW I can see similar test which will probably fail too. Anyway, what needs to be done yet in order to merge PR?